tests/full/test.fmf

```diff
@@ -0,0 +1,21 @@
+summary: Run the full tmt test suite in an external system
+test: ./test.sh
+framework: beakerlib
+adjust:
+- when: distro != fedora
+  require+:
+  - library(epel/epel)
+require:
+- rsync
+- git-core
+- rpm-build
+- make
+- procps-ng
+- python3-docutils
+recommend:
+- guestfs-tools
```

Comment:
Why not using `require` here as well?

Reply:
Because CRB might not be enabled yet. And wow can't run 'prepare' to enable it, hence it can be installed only after the `dnf config-manager --set-enabled` line

Reply:
And same for python3-docutils

Reply:
Ah, I see. Understood.